web/docs/language/features.md

```diff
@@ -443,6 +443,149 @@ import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
   }
 ```
 
+## Job Queues
+
+If you have long running tasks that require reliable, retriable, and/or schedulable (possibly recurring) work to be performed which should not be part of the normal request-response cycle, you may need a `job`! Wasp allows you to register `job` handler functions to be executed by a job executor, whereby you can manually enqueue work from JavaScript on the server, or have it automatically invoked for you via a cron schedule.
+
+### Job Executors
+
+Job executors handle the scheduling, monitoring, and execution of our job functions. They are the primary abstraction we provide that allows you to write one `async` JavaScript `job` handler function, but have it executed in many different job queue contexts by changing a single line in your `.wasp` file.
+
+#### pg-boss
+
+We have selected [pg-boss](https://github.com/timgit/pg-boss/) as our first job executor to handle the low-volume, basic job queue workloads many web applications have (think sending emails, connecting to external HTTP APIs, etc.). By using PostgreSQL (and [SKIP LOCKED](https://www.2ndquadrant.com/en/blog/what-is-select-skip-locked-for-in-postgresql-9-5/)) as it's storage and synchronization mechanism, it allows us to provide many job queue pros without any additional infrastructure or complex management.
+
+### Basic job definition and usage
+
+To register a `job` in Wasp, simply add a declaration with a reference to an `async` function, like the following:
+
+```css title="main.wasp"
+job mySpecialJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js",
+    options: {=json { "retryLimit": 1 } json=} // optional
+  }
+}
+```
+
+Then, in your [Operations](/docs/language/features#queries-and-actions-aka-operations) or [setupFn](/docs/language/features#setupfn-extimport-optional), you can enqueue work to be done:
```

Comment:
Or in some other piece of their code! They are all at the end run by Operation or setupFn though, but just wanted to emphasize that they can run them from any NodeJS code of their own.